src/overlays/actors/ovl_Bg_Dblue_Movebg/z_bg_dblue_movebg.c

```diff
@@ -28,46 +59,829 @@ const ActorInit Bg_Dblue_Movebg_InitVars = {
     (ActorFunc)BgDblueMovebg_Draw,
 };
 
-#endif
+static Gfx* D_80A2B8AC[] = {
+    NULL,
+    object_dblue_object_DL_0069D8,
+    NULL,
+    NULL,
+    NULL,
+    NULL,
+    object_dblue_object_DL_004848,
+    object_dblue_object_DL_0061B8,
+    NULL,
+    NULL,
+    NULL,
+    NULL,
+};
+
+static s32 D_80A2B8DC[] = { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0x0600CAA0, 0 };
```

Comment:
This is used to initialize `unk_168`, which seems to be used as a display list, so this array should be an array of `Gfx*` and the values of the array should be changed accordingly